rest/importtest/import_test.go

```diff
@@ -3000,3 +3000,105 @@ func TestImportFilterTimeout(t *testing.T) {
 	timeoutErr := rest.WaitWithTimeout(&syncFnFinishedWG, time.Second*15)
 	assert.NoError(t, timeoutErr)
 }
+
+func TestImportRollback(t *testing.T) {
+
+	if base.UnitTestUrlIsWalrus() {
+		t.Skip("This test only works against Couchbase Server - needs cbgt and import checkpointing")
+	}
+
+	base.SetUpTestLogging(t, base.LevelDebug, base.KeyImport, base.KeyDCP)
+
+	ctx := base.TestCtx(t)
+	bucket := base.GetPersistentTestBucket(t)
+	defer bucket.Close(ctx)
+
+	rt := rest.NewRestTester(t, &rest.RestTesterConfig{
+		CustomTestBucket: bucket.NoCloseClone(),
+		PersistentConfig: false,
+	})
+
+	key := "importRollbackTest"
+	lastSeq := "0"
+	var checkpointPrefix string
+	// do some setup work in an anonymous function so that we can be sure to close the rest tester on unexpected error
+	func() {
```

Comment:
I think it is OK to double close a `RestTester` https://github.com/couchbase/sync_gateway/blob/master/rest/utilities_testing.go#L564

Reply:
I prefer the approach here - relying on double-close guards in RestTester seems like less robust test design.